h2o-py/h2o/utils/shared_utils.py

```diff
@@ -353,6 +358,144 @@ def slice_is_normalized(s):
 handle_python_lists = _handle_python_lists
 check_lists_of_lists = _check_lists_of_lists
 
+gen_model_file_name = "h2o-genmodel.jar"
+h2o_predictor_class = "water.util.H2OPredictor"
+
+def find_file(name, path):
+    """
+    Helper function for predict_json() function to check if both MOJO and h2o-genmodel.jar are available
```

Comment:
this function is not specific to MOJOs, please fix the documentation

Reply:
In its current form, this function searches for file `name` within directory `path` or any of its subdirectories.
If you want this method to be used for `predict_json()` only, then it should start with underscore marking the function as private.